jest.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  setupTestFrameworkScriptFile: '<rootDir>/test/setup/setupEnzyme.js',
```

Comment:
what do you think about storing tests near files/components that they cover? imho it will be easier to read and maintain